cif_core.dic

```diff
@@ -25109,21 +25109,142 @@ save_atom_type_scat.versus_stol_list
          '_atom_type_scat_versus_stol_list'
          '_atom_type.scat_versus_stol_list'
 
-    _definition.update            2013-04-17
+    _definition.update            2023-04-10
```

Comment:
Maybe we should deprecate this item in favour of the new category?

Reply:
Ok, I deprecated the item in favour of those from the `ATOM_TYPE_SCAT_TABLE` category based on @jamesrhester's comment (https://github.com/COMCIFS/cif_core/pull/371#issuecomment-1600302774).